tests/unit/dataactcore/factories/staging.py

```diff
@@ -218,6 +218,59 @@ class Meta:
     uri = fuzzy.FuzzyText()
 
 
+class DetachedAwardFinancialAssistanceFactory(factory.Factory):
+    class Meta:
+        model = stagingModels.DetachedAwardFinancialAssistance
+
+    detached_award_financial_assistance_id = None
+    submission_id = fuzzy.FuzzyInteger(9999)
+    job_id = fuzzy.FuzzyInteger(9999)
+    row_number = fuzzy.FuzzyInteger(9999)
+    action_date = fuzzy.FuzzyText()
+    action_type = fuzzy.FuzzyText()
```

Comment:
Do we want to add default values for `action_type` also? The acceptable values should be [A, B, C, D, blank] but not sure if we need to get that deep for the default values for these.

Asked because I saw `record_type` was limited to 1 or 2.